utils-internal/build.gradle

```diff
@@ -16,4 +16,10 @@ kotlin {
             }
         }
     }
+
+    targets.configureEach {
+        compilations.configureEach {
+            compilerOptions.configure((Action) { it.freeCompilerArgs.add("-Xexpect-actual-classes") })
```

Comment:
What does it do?
Put it in `mpp-configuration` plugin?

Reply:
Will do, thanks.

Not the compilers emits a warning saying that expect/actual classes are Beta. This code disables the warning.

Reply:
Done.